plugins/com.aptana.samples.ui/src/com/aptana/samples/ui/portal/actionController/SamplesActionController.java

```diff
@@ -111,6 +120,143 @@ public Object getSamples()
 	}
 
 	/**
+	 * Adds a new sample from the local file system or a remote URI into the Samples Manager<br>
+	 * The attributes passed to this call should hold the following required attributes of the sample to be added.
+	 * <ul>
+	 * <li>category</li>
+	 * <li>name</li>
+	 * <li>id</li>
+	 * <li>description</li>
+	 * <li>location</li>
+	 * </ul>
+	 *
+	 * <pre>
+	 *   <b>Sample JS code:</b>
+	 *   <code>result = dispatch($H({controller:'portal.samples', action:"addSample", args:"sample_json_data"}).toJSON());</code>
```

Comment:
Would it be possible to give a example of the parameters we need to pass, in order to create a sample - such as location, remote/local etc.